react/components/VersionSelector.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FunctionComponent } from 'react'
+import { Dropdown } from 'vtex.styleguide'
+
+import { useAppVersionState, useAppVersionDispatch } from './AppVersionContext'
+
+const VersionSelector: FunctionComponent = () => {
+  const { major, availableMajors } = useAppVersionState()
+  const setMajorInfo = useAppVersionDispatch()
+
+  return (
+    <div className="w-40 flex">
+      <Dropdown
+        options={getDropdownOptions(availableMajors)}
```

Comment:
Use useMemo instead to generate those options

Reply:
Niice! Updated it